roles/java/tasks/main.yml

```
@@ -60,9 +60,14 @@
     name: adoptopenjdk11
     state: present
 
-- name: Detect JAVA_HOME
+- name: Register JAVA_HOME for java 8
   shell: /usr/libexec/java_home -v 1.8
-  register: detect_java_home
+  register: detect_java_8_home
```

i don't see why these need registering

I know it was here before, but why are we ignoring errors?

---

the registering is needed for jenv so we can set it up ready to be used